group_vars/rhcs.yml.sample

```diff
@@ -29,7 +29,7 @@ dummy:
 #  mimic: 13
 
 # Directory to fetch cluster fsid, keys etc...
-fetch_directory: ~/ceph-ansible-keys
+fetch_directory: "{{ inventory_dir }}/fetch/"
```

Comment:
this must remain `~/ceph-ansible-keys`, values are getting pickef from `rhcs_edits.txt`

Reply:
@leseb also updated ``rhcs_edits.txt`` so changes from ``inventory/sample/group_vars/rhcs.yml`` is valid as generated result from ``generate_group_vars_sample.sh``

Reply:
@leseb shall this still keep as ``~/ceph-ansible-keys``?

``rhcs_edits.txt`` also updated accordingly.